app/src/main/java/org/mozilla/fenix/library/history/HistoryItemTimeGroup.kt

```diff
@@ -4,20 +4,23 @@
 
 package org.mozilla.fenix.library.history
 
-import android.content.Context
+import android.content.res.Resources
 import org.mozilla.fenix.R
 import java.util.Calendar
 import java.util.Date
 
 enum class HistoryItemTimeGroup {
     Today, Yesterday, ThisWeek, ThisMonth, Older;
 
-    fun humanReadable(context: Context): String = when (this) {
-        Today -> context.getString(R.string.history_today)
-        Yesterday -> context.getString(R.string.history_yesterday)
-        ThisWeek -> context.getString(R.string.history_7_days)
-        ThisMonth -> context.getString(R.string.history_30_days)
-        Older -> context.getString(R.string.history_older)
+    /**
+     * Converts a [HistoryItemTimeGroup] enum to a matching string resource.
+     */
+    fun humanReadable(resources: Resources): String = when (this) {
```

Comment:
what's the context for the switch from `Context` to `Resources`?

Reply:
I had to reasons in mind. `Resources` is the real dependency here, as `context.getString` calls `getResources.getString` internally. And also, since this class is being used to build objects inside `HistoryDataSource`, I really didn't want to pass a `context` instance there.